Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/SessionManager.cs

```diff
@@ -0,0 +1,403 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Linq will allocate a bunch and is usually not friendly to game dev. Any reason we're using it?

Reply:
It is for this check `m_Portal.NetManager.ConnectedClientsIds.Contains(id)` in the `OnGameEnded` method. I'll look for an alternative